<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-12-03T12:46:13 IST" name="Default suite" finished-at="2020-12-03T12:46:27 IST" duration-ms="14001">
    <groups>
      <group name="smokeTest">
        <method signature="TC001_login.login()[pri:0, instance:testcases.TC001_login@49b0b76]" name="login" class="testcases.TC001_login"/>
      </group> <!-- smokeTest -->
    </groups>
    <test started-at="2020-12-03T12:46:13 IST" name="Default test" finished-at="2020-12-03T12:46:27 IST" duration-ms="14001">
      <class name="testcases.TC001_login">
        <test-method is-config="true" signature="beforeSuite()[pri:0, instance:testcases.TC001_login@49b0b76]" started-at="2020-12-03T12:46:13 IST" name="beforeSuite" finished-at="2020-12-03T12:46:13 IST" duration-ms="462" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method is-config="true" signature="setData()[pri:0, instance:testcases.TC001_login@49b0b76]" started-at="2020-12-03T12:46:13 IST" name="setData" finished-at="2020-12-03T12:46:13 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setData -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testcases.TC001_login@49b0b76]" started-at="2020-12-03T12:46:13 IST" name="beforeMethod" finished-at="2020-12-03T12:46:19 IST" duration-ms="5333" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="login()[pri:0, instance:testcases.TC001_login@49b0b76]" started-at="2020-12-03T12:46:19 IST" name="login" finished-at="2020-12-03T12:46:21 IST" duration-ms="1994" status="FAIL">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[FAILED]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: FAILED
at utilities.Reporter.reportStep(Reporter.java:74)
at wrappers.GenericWrapper.webElementClick(GenericWrapper.java:189)
at pages.LoginPage.clickLogin(LoginPage.java:41)
at modules.ReusableModule.login(ReusableModule.java:24)
at testcases.TC001_login.login(TC001_login.java:29)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testcases.TC001_login@49b0b76]" started-at="2020-12-03T12:46:21 IST" name="afterMethod" finished-at="2020-12-03T12:46:21 IST" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testcases.TC001_login@49b0b76]" started-at="2020-12-03T12:46:21 IST" name="beforeMethod" finished-at="2020-12-03T12:46:26 IST" duration-ms="5852" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="login()[pri:0, instance:testcases.TC001_login@49b0b76]" started-at="2020-12-03T12:46:26 IST" name="login" finished-at="2020-12-03T12:46:27 IST" duration-ms="743" status="FAIL">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[FAILED]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: FAILED
at utilities.Reporter.reportStep(Reporter.java:74)
at wrappers.GenericWrapper.webElementClick(GenericWrapper.java:189)
at pages.LoginPage.clickLogin(LoginPage.java:41)
at modules.ReusableModule.login(ReusableModule.java:24)
at testcases.TC001_login.login(TC001_login.java:29)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testcases.TC001_login@49b0b76]" started-at="2020-12-03T12:46:27 IST" name="afterMethod" finished-at="2020-12-03T12:46:27 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="afterSuite()[pri:0, instance:testcases.TC001_login@49b0b76]" started-at="2020-12-03T12:46:27 IST" name="afterSuite" finished-at="2020-12-03T12:46:28 IST" duration-ms="521" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- testcases.TC001_login -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
